xarray/core/npcompat.py

```diff
@@ -96,3 +97,99 @@ def __array_function__(self, *args, **kwargs):
 
 
 IS_NEP18_ACTIVE = _is_nep18_active()
+
+
+if LooseVersion(np.__version__) >= "1.20.0":
+    sliding_window_view = np.lib.stride_tricks.sliding_window_view
+else:
+    from numpy.core.numeric import normalize_axis_tuple
```

Comment:
```suggestion
    from numpy.core.numeric import normalize_axis_tuple  # type: ignore
```
there was a second one...